hackIDE/static/hackIDE/js/custom.js

```diff
@@ -411,6 +411,17 @@ $(document).ready(function(){
 							$(".error-message").html(response.run_status.stderr);
 						}
 					}
+					// To handle case when hackerearth api doesn't return a valid response.
```

Comment:
I put it in the success callback as the ajax call does succeed with a 200 OK status. It's just that it doesn't return a valid response. I can add the code in the failure call back as well if you want.